### DISCUSSION TITLE
lotus-miner - cannot get id from non id address

### DISCUSSION THREAD

**Discussion by JshFeel** [filecoin-project/lotus#7893] [Storage Provider] [answered]
### Lotus Version

```text
Daemon:  1.13.3-dev+mainnet+git.6f316cddb+api1.4.0
Local: lotus version 1.13.3-dev+mainnet+git.6f316cddb
```

### Describe the Bug

There was a problem while mining. So, I format the HDD and prepare for mining again.
But it keeps failing. You cannot mine again with the mining address you were mining.
Why did this happen to me...
I want to continue mining. It is already the 3rd mining address. I don't want to throw away this mining address. help me

### Logging Information

```text
ERROR: creating node: starting node: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): failed to build *miner.Miner: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): failed to build gen.WinningPoStProver: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): failed to build dtypes.MinerID: received non-nil error from function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): cannot get id from non id address
```

### Repo Steps

1. lotus-miner init --actor <f2~address>
2. success & lotus-miner run
3.  ERROR
ERROR: creating node: starting node: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): failed to build *miner.Miner: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): failed to build gen.WinningPoStProver: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): failed to build dtypes.MinerID: received non-nil error from function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:30): cannot get id from non id address

**Comment by Reiers** [comment]
Hi @JshFeel 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by LexLuthr** [comment, marked as answer]
@JshFeel Please check backup and restore for lotus miner.

https://lotu.sh/docs/storage-providers/backup-and-restore/

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
